tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -15,9 +15,9 @@
 #   limitations under the License.
 
 import os
-import xml.etree.ElementTree as ET
+from xml.etree import ElementTree
 from xml.dom.minidom import parseString
-import lxml.etree as etree
+from lxml import etree
```

Comment:
```suggestion
import lxml.etree
```

Reply:
note, you will need to update the usages:
[in bq_to_xml.py#L57](https://github.com/GoogleCloudPlatform/professional-services/blob/8f06b81c80b0c59ddfc927bdea1abe09061aa2d0/tools/bigquery-to-xml/bq_to_xml.py#L57)
to:
```python3
    doc = lxml.etree.XML(my_xml)
```
and [in bq_to_xml.py#L77](https://github.com/GoogleCloudPlatform/professional-services/blob/8f06b81c80b0c59ddfc927bdea1abe09061aa2d0/tools/bigquery-to-xml/bq_to_xml.py#L77)
```python3
    lxml.etree.strip_tags(doc, "deleteme")
```